cli/githubutil.py

```diff
@@ -176,6 +186,48 @@ def release_and_prepare_next_dev_cycle(
         commit_message="Prepare next dev cycle " + next_version_dev
     )
 
+def draft_release_name_for_version(release_version: str):
+    return "{v}-draft".format(v=release_version)
+
+def create_or_update_draft_release(
+    github_cfg_name: str,
+    github_repository_owner: str,
+    github_repository_name: str,
+    repository_branch: str,
+    release_version: str,
+    repo_dir: str=None
+):
+    github_cfg = ctx().cfg_factory().github(github_cfg_name)
+
+    helper = GitHubRepositoryHelper(
+        github_cfg=github_cfg,
+        owner=github_repository_owner,
+        name=github_repository_name,
+        default_branch=repository_branch,
```

Comment:
nit: I'd prefer the comma after the final line everywhere but either way is fine

Reply:
oh, was probably a copy paste thing as I usually do not add a comma after the last argument :-D
I would rather not change it everywhere with this PR and rather do it in a seperate one, if this is something that you generally do in this project

Reply:
Unfortunately we don't have a defined style here. Just leave it as it is then, we have an issue concerning code-style in our backlog.